dashboard/src/main/resources/templates/macros.ftl

```diff
@@ -8,23 +8,25 @@
   <#return plural?string(pluralNoun, singularNoun)>
 </#function>
 
-<#macro formatJarLinkageReport jar problemsToClassFiles jarToDependencyPaths dependencyPathRootCauses>
-  <!-- problemsToClassFiles: ImmutableMap<SymbolProblem, Set<String>> -->
-  <#if problemsToClassFiles?size gt 0>
+<#macro formatJarLinkageReport jar problemsWithClass jarToDependencyPaths dependencyPathRootCauses>
+  <!-- problemsToClasses: ImmutableSetMultimap<SymbolProblem, String> converted to
+    ImmutableMap<SymbolProblem, Collection<String>>; Multimaps don't work well with Freemarker -->
```

Comment:
What's the problem with Freemarker? Is it that `<#list multimap as key, value>` doesn't work?

Reply:
OK, so the comment that "Multimaps don't work well with Freemarker" isn't accurate. Can you remove it?

Reply:
Updated.